src/apps/token-atlas/components/TokenGraphColumn/TokenGraphColumn.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+
+// images
+import LogoEx from '../../images/token-atlas-logo-small.svg';
+import ArrowGreen from '../../images/arrow-circle-green.svg';
+// import ArrowRed from '../../images/arrow-circle-red.svg';
+// import ArrowRedSmall from '../../images/arrow-circle-red-small.svg';
```

Comment:
Just a reminder to remove unless you may need it in future

Reply:
I will need in the future, need the data to be dynamic to fully implement those